src/pages/ScholarshipDetails.jsx

```diff
@@ -10,18 +11,20 @@ function ScholarshipDetailsPage() {
     return (<h1>Scholarship Not Found</h1>);
   }
 
-  const [name, amount, deadline, website, school, year, description] = scholarship;
```

Comment:
Ohh was this the bug??

Reply:
Yes, The {} were needed for destructuring.

Reply:
Dang, the reason I had used `[]` was to avoid creating the new lines. Perhaps we can set **object-curly-newline.ObjectPattern** to `"never"` in the ESLint config so it allows this to be a single line with curly braces? I'm looking at
https://eslint.org/docs/rules/object-curly-newline#objectexpression-and-objectpattern

Reply:
Oh yeah I was thinking of using that